test/functional/p2p_feefilter.py

```diff
@@ -61,7 +54,9 @@ def set_test_params(self):
         # mempool and wallet feerate calculation based on GetFee
         # rounding down 3 places, leading to stranded transactions.
         # See issue #16499
-        self.extra_args = [["-minrelaytxfee=0.00000100", "-mintxfee=0.00000100"]] * self.num_nodes
+        # whitelist all peers to speed up tx relay / mempool sync
```

Comment:
Sorry I didn't notice this earlier; I think we're now starting to call this s/whitelist all/grant noban permission to/ (or something similar)

Reply:
Thanks, done.